### PR TITLE
Add raw and jpeg

### PR DESCRIPTION
**Pull request by joaoantoniocardoso**
I'll rebase to master after #155

### CODE REVIEW

**Comment by patrickelectric** [comment]
@joaoantoniocardoso please rebase

File: src/stream/pipeline/v4l_pipeline.rs [hunk 1]
```diff
@@ -74,9 +74,43 @@ impl V4lPipeline {
...
+            ),
+            device = device,
+            width = width,
+            height = height,
+            interval_denominator = interval_denominator,
+            interval_numerator = interval_numerator,
+            filter_name = PIPELINE_FILTER_NAME,
+            tee_name = PIPELINE_TEE_NAME
+            )
+            }
```
Comment (patrickelectric) [inline src/stream/pipeline/v4l_pipeline.rs:93]: the code format looks weird

Comment (joaoantoniocardoso) [reply src/stream/pipeline/v4l_pipeline.rs:93]: Yeah, I didn't like it either, but couldn't be different :(

Comment (joaoantoniocardoso) [reply src/stream/pipeline/v4l_pipeline.rs:95]: cargo fmt was just failing to automatically get it right but fixing it by hand works :) Thanks!

File: src/stream/pipeline/v4l_pipeline.rs [hunk 2]
```diff
@@ -49,34 +49,71 @@ impl V4lPipeline {
...
+                    filter_name = PIPELINE_FILTER_NAME,
+                    tee_name = PIPELINE_TEE_NAME
+                )
+            }
+            VideoEncodeType::MJPG => {
+                format!(
+                    concat!(
+                        "v4l2src device={device} do-timestamp=false",
+                        " ! jpegenc quality=85 idct-method=1",
+                        " ! capsfilter name={filter_name} caps=video/x-jpeg,width={width},height={height},framerate={interval_denominator}/{interval_numerator}",
```
Comment (joaoantoniocardoso) [inline src/stream/pipeline/v4l_pipeline.rs:101]:
```suggestion
                        " ! capsfilter name={filter_name} caps=image/jpeg,width={width},height={height},framerate={interval_denominator}/{interval_numerator}",
```

**Review by patrickelectric — APPROVED** [review]
Be free to merge after corrections

**Pull request merged**